process/smartContract/process.go

```diff
@@ -1600,11 +1604,16 @@ func (sc *scProcessor) createSCRForSenderAndRelayer(
 	relayedSCR, isRelayed := isRelayedTx(tx)
 	shouldRefundGasToRelayerSCR := isRelayed && callType != vmcommon.AsynchronousCall && gasRefund.Cmp(zero) > 0
 	if shouldRefundGasToRelayerSCR {
+		senderForRelayerRefund := tx.GetRcvAddr()
```

Comment:
can we have instead some hardcoded (constant) sender addresses for relayed txs that are constructed for each shard?